auraed/src/runtime/error.rs

```diff
@@ -2,27 +2,40 @@ use log::error;
 use thiserror::Error;
 use tonic::Status;
 
-#[derive(Error, Debug, PartialEq, Eq)]
-pub enum CellServiceError {
+pub(crate) type Result<T> = std::result::Result<T, RuntimeError>;
+
+#[derive(Error, Debug)]
+pub(crate) enum RuntimeError {
     #[error("missing argument in request: {arg}")]
     MissingArgument { arg: String },
```

Comment:
We can take this out as validation takes care of it.

Reply:
i tried, but i needed this to go from ValidationError to RuntimeError to Status.

i didn't see anything that would convert ValidationError straight to Status.. did i miss it?